src/openrct2-ui/windows/GameBottomToolbar.cpp

```diff
@@ -421,8 +421,8 @@ static void window_game_bottom_toolbar_draw_left_panel(rct_drawpixelinfo* dpi, r
     if (!(gParkFlags & PARK_FLAGS_NO_MONEY))
     {
         rct_widget widget = window_game_bottom_toolbar_widgets[WIDX_MONEY];
-        int32_t x = w->x + (widget.left + widget.right) / 2;
-        int32_t y = w->y + (widget.top + widget.bottom) / 2 - (line_height == 10 ? 5 : 6);
+        int32_t x = w->windowPos.x + (widget.left + widget.right) / 2;
+        int32_t y = w->windowPos.y + (widget.top + widget.bottom) / 2 - (line_height == 10 ? 5 : 6);
```

Comment:
Note to self or @AaronVanGeffen : shouldn't we just divide `line_height` by 2 here?

Reply:
No, some fonts use a line height of 14 or 15 while the glyphs are offset to compensate. It's a little messy, but the main reason why I hardcoded this.